### PR TITLE
[sdk 309] View Repo removed

### PR DESCRIPTION
**Pull request by ZahidZafar**

### CODE REVIEW

File: Assets/Plugins/CountlySDK/Helpers/CountlyStorageHelper.cs [hunk 1]
```diff
@@ -107,6 +121,10 @@ internal void RunMigration()
             // _schemaVersion = 1 : deletion of the data in the “EventNumberInSameSessionEntity” table
             if (_currentVersion == 0) {
                 Migration_EventNumberInSameSessionEntityDataRemoval();
+                Migration_CopyViewDataIntoEventData();
```
Comment (ArtursKadikis) [inline Assets/Plugins/CountlySDK/Helpers/CountlyStorageHelper.cs:124]: the comment above the "if" should mention the new changes

File: Assets/Plugins/CountlySDK/Helpers/CountlyStorageHelper.cs [hunk 2]
```diff
@@ -118,8 +136,25 @@ internal void RunMigration()
...
+        private void Migration_CopyViewDataIntoEventData()
+        {
+            while (ViewRepo.Count > 0) {
+                EventRepo.Enqueue(ViewRepo.Dequeue());
+            }
+            _logHelper.Verbose("[CountlyStorageHelper] Migration_CopyViewDataIntoEventData");
+
+        }
+
+        internal void DeleteAllTablesData() {
```
Comment (ArtursKadikis) [inline Assets/Plugins/CountlySDK/Helpers/CountlyStorageHelper.cs:154]: Rename this to "ClearDBData"

File: Assets/Plugins/CountlySDK/Persistance/Repositories/Impls/NonViewEventRepository.cs [hunk 3]
```diff
@@ -13,7 +13,7 @@ public NonViewEventRepository(Dao<EventEntity> dao, SegmentDao segmentDao, Count
 
         protected override bool ValidateModelBeforeEnqueue(CountlyEventModel model)
         {
-            return !model.Key.Equals(CountlyEventModel.ViewEvent);
+            return true;
```
Comment (ArtursKadikis) [inline Assets/Plugins/CountlySDK/Persistance/Repositories/Impls/NonViewEventRepository.cs:16]: Why this change?

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]

File: Assets/Plugins/CountlySDK/Persistance/Repositories/Impls/NonViewEventRepository.cs (hunk 3 above)
Comment (ZahidZafar) [reply Assets/Plugins/CountlySDK/Persistance/Repositories/Impls/NonViewEventRepository.cs:16]: this check preventing the event with key '[CLY]_view'  from adding to the event repository.

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
After the last changes you didn't even build the project to make sure it works.
Go on, try it, then you'll find your mistake.

**Review by ArtursKadikis — APPROVED** [review]
